app/services/create_user_service.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class CreateUserService
+  def initialize(params)
+    @params = params.dup
+  end
+
+  def run!
+    user_type = case @params[:userType]
+                when 'driver'
+                  User::Driver
+                when 'passenger'
+                  User::Passenger
+                else
+                  return
+                end
+    user_type.create!(
+      email: @params[:email],
+      password: @params[:password],
+      status: @params[:status] || 'normal'
```

Comment:
empty string 값인 경우에는 기본값이 사용되지 않기에 이러한 경우 `presence` 메소드를 사용해주세요.
```suggestion
      status: @params[:status].presence || 'normal'
```

Reply:
[8ac2153](https://github.com/travelingaries/taxi-dispatch-backend-api/pull/1/commits/8ac21532f29b96b40aa34cc6eb76f13128abeffd)에서 위 방법으로 수정했습니다 🙆‍♂️